lib/inspec-vault/input.rb

```diff
@@ -105,5 +143,54 @@ def fetch_plugin_setting(setting_name, default = nil)
     def fetch_vault_setting(setting_name)
       ENV[setting_name.upcase] || plugin_conf[setting_name]
     end
+
+    # Check if this is called from within TestKitchen
+    def inside_testkitchen?
+      !! defined?(::Kitchen)
+    end
+
+    # Access to kitchen data
+    # TODO: Switch to official API discussed in test-kitchen/test-kitchen#1674
+    def kitchen
+      require "binding_of_caller"
```

Comment:
It looks like we need to add a gemspec dependency from inspec-vault to binding_of_caller -it's not otherwise present in my testing.